test/extended/cluster/cl.go

```diff
@@ -309,6 +243,83 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 	})
 })
 
+func postWait(oc *util.CLI, namespaces []string) error {
+	// Wait for builds and deployments to complete
+	for _, ns := range namespaces {
+		rcList, err := oc.AdminKubeClient().CoreV1().ReplicationControllers(ns).List(metav1.ListOptions{})
+		if err != nil {
+			return err
```

Comment:
I wonder why the more specific error messages were dropped after the code was lifted into this function.  Are the generic err(s) that you return enough to tell the user what happened?

Reply:
Wanted to pass the errors back to be handled in the main function, passing these new errors instead. What do you think?